sw/device/lib/dif/autogen/meson.build

```diff
@@ -198,6 +198,20 @@ sw_lib_dif_autogen_otp_ctrl = declare_dependency(
   )
 )
 
+# Autogen Power Manager DIF library
```

Comment:
Strange to see this in pinmux PR.

Reply:
For some reason the diff renders weird. It wasn't added, just moved to be in alpha-order with other targets.

Reply:
Yeah, I also was confused in an earlier PR, the diff is OK, it's just easy to miss the difference in the first hunk.